ci: Remove -new suffix from helm directory

camscale opened the pull request:
Remove the `-new` suffix from the helm plugins directory and the
associated TODO. That TODO was done in the buildbox in commit
5d53c91c7a8d41391422171749b3a3d7f23084ab.

`make test-helm` is now failing on CI due to that removal, so followup
with this removal.

camscale committed:
ci: Remove -new suffix from helm directory

Remove the `-new` suffix from the helm plugins directory and the
associated TODO. That TODO was done in the buildbox in commit
5d53c91c7a8d41391422171749b3a3d7f23084ab.

`make test-helm` is now failing on CI due to that removal, so followup
with this removal.

camscale commented:
Well, `helm unittest` runs now, but there is a different error: `-3` seems to be an invalid flag

r0mant reviewed (commented):
I think you may also need to remove `-3` flag from `helm unittest` commands in the Makefile. cc @hugoShaka

camscale committed:
Remove -3 from helm unittest

Remove the `-3` flag from the invocation of `helm unittest` in the
`Makefile`. The updated buildbox appears to not support this (deprecated
flag now gone?)

r0mant reviewed (changes requested):
@camscale Actually, the correct fix is to bring the proper helm unittest plugin that was removed in https://github.com/gravitational/teleport/commit/5d53c91c7a8d41391422171749b3a3d7f23084ab back.

camscale commented:
Closing this in favour of adding the removed helm version back to the buildbox
